client/app/components/layout.coffee

```diff
@@ -7,7 +7,7 @@ Menu                  = React.createFactory require './menu/menu'
 Modal                 = React.createFactory require './modal'
 ToastContainer        = React.createFactory require './toast_container'
 Tooltips              = React.createFactory require './tooltips-manager'
```

Comment:
So… `tooltips-manager` or `tooltips_manager` x) ? Just kidding, let's take it as it ;)